roles/os-keystone/README.md

```diff
@@ -1,30 +1,127 @@
-# Ansible role: OpenStack-Keystone
+# os-keystone
+This role setups the OpenStack Identity service.
 
-This role setups the OpenStack Identity service
+## Requirements
+- A previously setup database server.
+  You can setup one with `mariadb` role
```

Comment:
[same comment](https://github.com/clearlinux/clear-config-management/pull/20/files#r70143074) applies here (point to URL of the roles)